### DISCUSSION TITLE
How to reset the timer?

### DISCUSSION THREAD

**Discussion by fio-la** [vydimitrov/react-countdown-circle-timer#92] [General]
Hi,

I try to find a way to reset the timer to a duration from props, but somehow I can't make it work. 

Can you have a look? 

https://gist.github.com/fi-cyber/62f0ab76c3e9677c77bd01a2ebc33268

**Comment by fio-la** [comment]
Short: The Timer.js is the component and in the UebungenAnzeigen.js I am displaying the timer. Everythis works well (displaing) except reseting the timer.

**Comment by vydimitrov** [comment]
Check this one https://github.com/vydimitrov/react-countdown-circle-timer/discussions/90#discussioncomment-404768
